bbb-exporter/collector.py

```diff
@@ -129,3 +139,16 @@ def collect(self):
 
         logging.info("Finished collecting metrics from BigBlueButton API")
 
+    @staticmethod
+    def _get_participant_count_by_client(self, meetings):
```

Comment:
We get a TypeError when running the exporter because of this `self`. We do not need a self parameter since the function is static. Remove `self` from the function prototype.